docs/data/migration/migration-tree-view-v6/migration-tree-view-v6.md

```diff
@@ -329,6 +329,34 @@ you can use the new `onNodeSelectionToggle` prop which is called whenever a node
 
 :::
 
+### Focus the Tree Item instead of the Tree View
+
+The focus is now applied to the Tree Item root element instead of the Tree View root element.
+This will mostly impact how you write tests to interact with the Tree View:
+
+For example, if you were writing a test with `react-testing-library`, here is what the changes could look like:
+
+```diff
+ it('test example on first item', () => {
+-  const { getByRole } = render(
++  const { getAllByRole } = render(
```

Comment:
In the test we tend to use `getByTestId`, but we could probably use `getAllByRole` and remove all the test ids.